ros_ign_interfaces/msg/Light.msg

```diff
@@ -0,0 +1,29 @@
+std_msgs/Header header
+
+string name
+
+# Light type: constant definition
+uint8 POINT       = 0
+uint8 SPOT        = 1
+uint8 DIRECTIONAL = 2
+
+uint8 type
+
+geometry_msgs/Pose pose
```

Comment:
We should document all message fields. For example:

https://github.com/ignitionrobotics/ros_ign/blob/48d4abb3be9278d15328ed5355f7aef48879df69/ros_ign_interfaces/msg/Contact.msg#L1-L6